src/index.html

```diff
@@ -16,17 +16,34 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <form
+      name="search-try"
+      action="/submit"
+      method="post"
+      class="search-block"
+      data-qa="big"
+    >
+      <input
+        class="input-bg input-bg--hover input-bg--focus search-block__item"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form
+      name="search-try2"
+      action="/submit"
+      method="post"
+      class="search-block"
+      data-qa="small"
```

Comment:
The `data-qa` attribute for the second form is set to `small`, which does not meet the task requirements. Please ensure that the `data-qa` attributes are correctly set as per the task description.